spec/templates/markup_processor_integrations/integration_spec_helper.rb

```diff
@@ -26,7 +26,7 @@
 
   before(:each) do
     if html_renderer.markup_class(markup).nil?
-      skip "Missing markup renderer #{markup}"
+      raise "Missing markup renderer #{markup}"
```

Comment:
This should stay as skip for systems that do not have the dependencies installed (testing outside of bundle exec).

Reply:
I am unable to run `rake`/`rspec` locally without a prior `bundle install`.